src/AutoMerge/guidelines.jl

```diff
@@ -32,7 +32,9 @@ function meets_compat_for_julia(working_directory::AbstractString, pkg, version)
     package_relpath = get_package_relpath_in_registry(;
         package_name=pkg, registry_path=working_directory
     )
-    compat = Pkg.TOML.parsefile(joinpath(working_directory, package_relpath, "Compat.toml"))
+    compat_file = joinpath(working_directory, package_relpath, "Compat.toml")
+    assert_allowed_to_not_exist(compat_file)
```

Comment:
I don't really understand this `assert`. It will never fire because you are calling it with a compat file just defined above. The act of calling `maybe_parse_toml` means that the file is allowed to not exist. So I am not sure the `assert` adds anything?

Reply:
I've removed the asserts.